### PR TITLE
Add ionic core init

### PR DESCRIPTION
**Pull request by michaeltintiuc**
Initialize the ionic core and load the required CSS and SVGs. Currently we have to manually add the icons which is extremely tedious below are some suggestions for improvement

Ways to improve:
- Collaborate with Ionic folks to add an auto-generated index.js files in the dist/ionic/svg/ dir containing svg imports. This way we can stay flexible and keep ionic a peer dependency
- Instead of using a file-loader in webpack with a regex of /(ios|md|logo)-\.(svg)(\?.*)?$/ which I find to be very vague, we can see how Vuejs handles it and create our own ionic-loader that would aprse all ion-icon elements and import only the required SVGs

For example svg/index.js would be imported as: `import '@ionic/core/dist/ionic/svg'`
And would contain:
```js
export * from './md-arrow-back.svg'
export * from './ios-arrow-back.svg'
export * from './md-arrow-forward.svg'
export * from './ios-arrow-forward.svg'
export * from './md-close.svg'
export * from './ios-close.svg'
...
```

### CODE REVIEW

**Comment by jmalfatto** [comment]
# [Codecov](https://codecov.io/gh/ModusCreateOrg/ionic-vue/pull/22?src=pr&el=h1) Report
> Merging [#22](https://codecov.io/gh/ModusCreateOrg/ionic-vue/pull/22?src=pr&el=desc) into [master](https://codecov.io/gh/ModusCreateOrg/ionic-vue/commit/10c4750bb2f9c0465be53156ab8718d1027ec459?src=pr&el=desc) will **not change** coverage.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/ModusCreateOrg/ionic-vue/pull/22?src=pr&el=tree)

```diff
@@          Coverage Diff          @@
##           master    #22   +/-   ##
=====================================
  Coverage     100%   100%           
=====================================
  Files           6      6           
  Lines         155    155           
  Branches       36     36           
=====================================
  Hits          155    155
```


| [Impacted Files](https://codecov.io/gh/ModusCreateOrg/ionic-vue/pull/22?src=pr&el=tree) | Coverage Δ | |
|---|---|---|
| [src/router.js](https://codecov.io/gh/ModusCreateOrg/ionic-vue/pull/22/diff?src=pr&el=tree#diff-c3JjL3JvdXRlci5qcw==) | `100% <ø> (ø)` | :arrow_up: |

------

[Continue to review full report at Codecov](https://codecov.io/gh/ModusCreateOrg/ionic-vue/pull/22?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/ModusCreateOrg/ionic-vue/pull/22?src=pr&el=footer). Last update [10c4750...2457531](https://codecov.io/gh/ModusCreateOrg/ionic-vue/pull/22?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by grgur** [comment]
Does this change warrant updates in the Readme and anywhere else `unpkg` is referenced?

**Comment by michaeltintiuc** [comment]
@grgur yes and now, if for some reason ionic is used in a browser, like how the cookbook is, then unpkg remains, but absolutely should update the README too, will do so

File: src/ionic.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import '@ionic/core/css/ionic.css'
+import 'ionicons/dist/collection/icon/icon.css'
+import '@ionic/core/dist/ionic/svg'
+import { defineCustomElements } from '@ionic/core/dist/esm'
+
+export default {
+  init(opts = {}) {
+    defineCustomElements(window, opts)
```
Comment (grgur) [inline src/ionic.js]: what if this is SSR? then `window` doesn't exist

Comment (michaeltintiuc) [reply src/ionic.js]: That is a great and very valid point and I believe we should bring this up with Ionic as this is how they're initializing as well AFAIK:
https://github.com/ionic-team/ionic/blob/master/angular/src/app-initialize.ts#L9

**Comment by michaeltintiuc** [comment]
Do not increment version as there are still upstream dependencies

**Pull request merged**